### DISCUSSION TITLE
Hyperion: (ERROR) Lock/unlock handler - Session bus is not connected

### DISCUSSION THREAD

**Discussion by jkserial** [hyperion-project/hyperion.ng#1657] [Q&A] [answered]
Okay I am trying to setup an Ambilight situation with Hyperion (first time using a Rasberry Pi). I have everything installed RPI Light and Hyperion is set to root. I believe the only thing holding me up at this point is that I keep getting the error (ERROR) Lock/unlock handler- Session bus is not connected when I do the command, "Sudo Systemctl Status Hyperion@Root.service" I really believe that this is the holdup to my success. Pleaseany advice would be amazing.

**Comment by Lord-Grey** [comment, marked as answer]
Hyperion is able to listen to screen locking/unlocking events, if the underlying operating system supports this.
Your OS does not support that feature why originally
Error: Lock/unlock handler- Session bus is not connected"
is logged.

As this is not an error and confused users including yourself, the message was rephrased to
Info: The lock/unlock feature is not supported by your system configuration
to inform the user that the lock/unlock feature will not be available.

Hoping that clarifies. It is not an Error.
